### ISSUE TITLE
Streamline the course by removing pieces students don't need to know - Template's section

### ISSUE THREAD

**Issue by reverest66** [chef-training/chef-essentials#162]
We spend a lot of time in this module talking about what students shouldn't use with respect to the resource (cookbook_file, remote_file, template).  I think that we only need to show the students how to use docs.io once, and we do that much earlier in the course.  In this case, let's talk about the template resource, and not convolute it by talking about the other two resources.  It just muddies the water for the students.

**Comment by ehaselwanter** [comment]
agreed. I was once again super confused myself in this part of the training. there should be just good enough motivation why we transition from file to template
